ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -70,6 +70,13 @@
           </td>
         </tr>
 
+        <tr v-else-if="!isMhrTransfer && row.index === 0 && hasMixedOwnersInGroup(row.item.groupId)">
+          <HomeOwnersMixedRolesError
+            :groupId="row.item.groupId"
+            :borderError ='isInvalidOwnerGroup(row.item.groupId)'
```

Comment:
For boolean props, it's better to use a verb in the name, eg. `is...` or `has...` or in this case `showBorderError`